Libraries/JavaScriptAppEngine/polyfills/document.js

```diff
@@ -1,34 +1,32 @@
-/* eslint global-strict: 0 */
-(function(GLOBAL) {
-  /**
-   * The document must be shimmed before anything else that might define the
-   * `ExecutionEnvironment` module (which checks for `document.createElement`).
-   */
+/* eslint strict: 0 */
+/**
+ * The document must be shimmed before anything else that might define the
+ * `ExecutionEnvironment` module (which checks for `document.createElement`).
+ */
```

Comment:
Do we have a problem with a specific version of the `ExecutionEnvironment` module?

https://github.com/facebook/fbjs/blob/master/src/core/ExecutionEnvironment.js#L14-L18 seems to do the right thing, i.e. it doesn’t strictly need `document` to exist.